test_package/conanfile.py

```diff
@@ -6,6 +6,9 @@ class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake"
 
+    def requirements(self):
+        self.requires("cmake/3.19.3")
+
```

Comment:
```suggestion
```
We usually don't reference cmake in the (build_)requirements of a recipe, because it is already provided by the system. Does gstreamer 1.18.0 need a specific cmake version ?

Reply:
I've specify it on the recipe cause it was missing on my windows. I have not done the vcvarsall command before launching the build.
So I've taken the last version by default. 

But if it's not a "good practice" yes you should remove it !

Reply:
FYI, you can add this to your profile:
```
[build_requires]
cmake/3.19.3
```
and it will be available for all your recipes